.circleci/config.yml

```diff
@@ -0,0 +1,160 @@
+defaults: &defaults
+  docker:
+    - image: petrpliska/shopsys-buildpack:0.1.1
+      auth:
+        username: $DOCKER_USERNAME
+        password: $DOCKER_PASSWORD
+  working_directory: ~/source
+
+login-to-dockerhub: &login-to-dockerhub
+  name: Login to Docker hub
+  command: echo $DOCKER_PASSWORD | docker login --username $DOCKER_USERNAME --password-stdin
+
+version: 2
+jobs:
+  deploy-to-k8s:
+    <<: *defaults
+    steps:
+      - setup_remote_docker
+      - checkout
+      - run:
+           <<: *login-to-dockerhub
+      - run:
+          name: Build docker image
+          command: |
+            # Calculate Docker image tag from Dockerfile checksum. Docker Image tag is used later in Kubernetes deployment.
+            # On shopsys development server we can assume, that Docker image used by Kubernetes deployment will be always present, because it was builded on same server.
+            # In common we should push this image into Docker registry and use image from registry later.
+            DOCKER_IMAGE="${DOCKER_IMAGE:=shopsys/shopsys}"
+            DOCKER_IMAGE_TAG="$(cksum Dockerfile | cut -d ' ' -f1)"
```

Comment:
This is a workaround for CircleCi absence of docker cache for free plans. Without this CircleCi would build docker image on every build. This is not necessary yet, because the source code is attached via volume and docker image should be rebuilt only when Dockerfile changes